AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,10 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+kontrast.dk#$##modal-background { display: none !important; }
+kontrast.dk#$#.modal-backdrop { display: none !important; }
+kontrast.dk#$#body { overflow: visible !important; }
+sued.aldi-liefert.de,enirocentrum.pl,felix1.de,goldenline.pl,panoramafirm.pl,hofer.at,dizioneri.com,m.n11.com,wuerth.de,greenweez.com,deutsches-ausschreibungsblatt.de,sued.aldi-liefert.de,capveriant.com,calkalasutasayrani.com,xn--giant-dsseldorf-5vb.de##.modal-backdrop
```

Comment:
Also please check those domains. If they need to change the body attribute, you should remove some domains from the list.

Reply:
Not again.

Reply:
@zloyden Please evaluate this site. Another code `https://www.goldenline.pl/`

Reply:
It seems these rules are needed for `goldenline.pl`

```
goldenline.pl#$#body{ overflow: visible !important; padding-right: 0 !important; }
goldenline.pl#$?#.cookie-modal:upward(div#react-portals) { display: none !important; }
goldenline.pl#@#.cookie-modal
```

Reply:
Added.